modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/index.scss

```diff
@@ -1,111 +1,145 @@
-.page-activities {
-	.blue-line-activites:after {
-		background-color: #4c84ff;
-		bottom: -1px;
-		content: '';
-		display: block;
-		left: -1px;
-		position: absolute;
-		top: 0;
-		width: 5px;
-	}
+.border-link-active::after {
+	background-color: var(--primary);
+	content: '';
+	display: block;
+	height: 1px;
+	margin-top: 6px;
+	position: absolute;
+	width: 25px;
+}
 
-	.border-link-active::after {
-		background-color: var(--primary);
-		content: '';
-		display: block;
-		height: 1px;
-		margin-top: 6px;
-		position: absolute;
-		width: 25px;
-	}
+.blue-line-activites:after {
+	background-color: #4c84ff;
+	bottom: -1px;
+	content: '';
+	display: block;
+	left: -1px;
+	position: absolute;
+	top: 0;
+	width: 5px;
+}
 
-	.box-activites {
-		height: 53px;
-	}
+.box-activites {
+	align-items: center !important;
```

Comment:
Can this be done with bootstrap? align-items-center

Reply:
Done! Thx!